py-polars/polars/dataframe/frame.py

```diff
@@ -3538,6 +3563,9 @@ def write_delta(
         delta_write_options
             Additional keyword arguments while writing a Delta lake Table.
             See a list of supported write options `here <https://github.com/delta-io/delta-rs/blob/395d48b47ea638b70415899dc035cc895b220e55/python/deltalake/writer.py#L65>`__.
+        delta_merge_options
+            Keyword arguments which are required to `MERGE` a Delta lake Table.
+            See a list of supported merge options `here <https://github.com/delta-io/delta-rs/blob/d833f080367276003af0e2c73c658ff96e671913/python/deltalake/table.py#L595C14-L595C14`__.
```

Comment:
I would prefer that this be a link to the latest deltalake API reference, as the options may change over time as new versions release.

Reply:
I agree, the issue is that we are undergoing a change in our docs and the url. I will update once our mkdocs are released, and it should be soon :)